api/src/pcapi/scripts/affect_correct_beneficiary_roles/main.py

```diff
@@ -0,0 +1,56 @@
+"""
+Job console documentation here: https://www.notion.so/passcultureapp/Documentation-Job-Console-769beeacd5a146de9c97b6f8ee544276
+Assumed path to the script (copy-paste in github actions): 
+
+https://github.com/pass-culture/pass-culture-main/blob/pc-35213/api/src/pcapi/scripts/affect_correct_beneficiary_roles/main.py
+
+"""
+
+import argparse
+import datetime
+import logging
+
+from pcapi.app import app
+from pcapi.core.finance.models import Deposit
+from pcapi.core.finance.models import DepositType
+from pcapi.core.users.models import User
+from pcapi.core.users.models import UserRole
+from pcapi.models import db
+
+
+logger = logging.getLogger(__name__)
+
+
+def main(not_dry: bool) -> None:
+    query = User.query.joinedload(User.deposits).filter(
+        User.validatedBirthDate < datetime.date(2007, 3, 21),  # Only over 18yo users are concerned
```

Comment:
```suggestion
        User.validatedBirthDate < datetime.date.today() - relativedelta(years=15),  # Only over 18yo users are concerned
```
Même si, si tout se passe bien, le script tournera bien quand tu l'as prévu